docs/manual/docs/cmd/accesstoken-get.md

```diff
@@ -0,0 +1,38 @@
+# accesstoken get
+
+Gets access token for the specified resource
+
+## Usage
+
+```sh
+accesstoken get [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-r, --resource <resource>`|The resource for which to retrieve an access token
+`--new`|Retrieve a new access token to ensure that it's valid for as long as possible
```

Comment:
Rookie issue found! Should `new` be optional or mandatory like `--new [new]`?

Reply:
Nope, because it's a flag. When you specify `--new [new]` then you expect a value to be passed along like `--new ohyeah`. When you specify only `--new`, then it's a flag. See also flags like debug or verbose: https://github.com/pnp/office365-cli/blob/cd657ed0fbdd5bec319d846f2dce906390f28dc2/src/Command.ts#L154